gen/atlas.tmpl

```diff
@@ -12,81 +12,26 @@ permissions:
   pull-requests: write
 jobs:
   lint:
-    {{- if eq .Driver "mysql" }}
-    services:
-      # Spin up a mysql:8 container to be used as the dev-database for analysis.
-      mysql:
-        image: mysql:8
-        env:
-          MYSQL_DATABASE: dev
-          MYSQL_ROOT_PASSWORD: pass
-        ports:
-          - 3306:3306
-        options: >-
-          --health-cmd "mysqladmin ping -ppass"
-          --health-interval 10s
-          --health-start-period 10s
-          --health-timeout 5s
-          --health-retries 10
-    {{- else if eq .Driver "postgres" }}
-    services:
-      # Spin up a postgres:15 container to be used as the dev-database for analysis.
-      postgres15:
-        image: postgres:15
-        env:
-          POSTGRES_DB: dev
-          POSTGRES_PASSWORD: pass
-        ports:
-          - 5432:5432
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-start-period 10s
-          --health-timeout 5s
-          --health-retries 5
-    {{- else if eq .Driver "mariadb" }}
-    services:
-      # Spin up a mariadb:11 container to be used as the dev-database for analysis.
-      maria11:
-        image: mariadb:11
-        env:
-          MYSQL_DATABASE: dev
-          MYSQL_ROOT_PASSWORD: pass
-        ports:
-          - 3306:3306
-        options: >-
-          --health-cmd "mysqladmin ping -ppass"
-          --health-interval 10s
-          --health-start-period 10s
-          --health-timeout 5s
-          --health-retries 10
-    {{- end }}
+    {{- template "Services" . }}
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: ariga/atlas-action@v0
+      - uses: ariga/atlas-action@master
```

Comment:
use `v0` 
it's customary in github actions land to tag the latest stable release with vX